opm/material/fluidmatrixinteractions/EclEpsConfig.hpp

```diff
@@ -196,8 +196,9 @@ class EclEpsConfig
                       const Opm::EclipseState& eclState,
                       Opm::EclTwoPhaseSystemType twoPhaseSystemType)
     {
+        const auto& endscale = eclState.runspec().endpoint_scaling();
```

Comment:
Maybe - the goal is a better interface.

It'll surely not happen before Deck is private.